p2p/exchange.go

```diff
@@ -21,10 +21,15 @@ import (
 
 var log = logging.Logger("header/p2p")
 
-// the minimum number of headers of the same height received from trusted peers
-// to determine the network head. If all trusted header will return headers with
-// non-equal height, then the highest header will be chosen.
-const minTrustedHeadResponses = 2
+// minHeadResponses is the minimum number of headers of the same height
+// received from peers to determine the network head. If all trusted peers
+// will return headers with non-equal height, then the highest header will be
+// chosen.
+const minHeadResponses = 2
+
+// minUntrustedHeadRequests is the minimum number of head requests to be made to
+// the network in order to determine the network head.
+var minUntrustedHeadRequests = 4
```

Comment:
Looking into usage of `minUntrustedHeadRequests` it seems that it is not minimum, but exact amount of requests that will be made to untrusted peers.

Reply:
done https://github.com/celestiaorg/go-header/pull/53/commits/60bda34a66dc73030d49cfdcda2e197e770b5939